docs/getting-started/remote-machine.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started on Remote Machine
+
+## Dependencies
+
+* SSH server
+
+That’s it. There is no need to install Mainframer on the remote machine.
+
+## Users
+
+We recommend creating a OS-level user per person
+and using real names to help with distinguishing people.
+For example, a username can be in a format of `<FIRST_NAME>_<LAST_NAME>`.
+
+There are different options, like a Docker container per person,
+but we’ll leave it up to you to decide.
+
+### Linux
+
+Be advised that Mainframer runs SSH in non-interactive mode.
+Some default Bash `.bashrc` configurations stop evaluation in this mode.
+In such cases place custom environment variable declarations
+and script invocations before the following line.
+
+```bash
+# If not running interactively, don't do anything
+```
+
+We’ve built a Bash script which might help with creating Linux users.
+
+Usage:
+
+```console
+$ bash create-user.sh "USER_NAME" "USER_SSH_PUBLIC_KEY"
```

Comment:
no need for `bash`, script has proper shebang line already, just `./create-user.sh …`

Reply:
Well, you have to make it executable beforehand, which will result in another command here.